src/code/bank2.asm

```diff
@@ -6317,6 +6326,7 @@ LinkDirectionCompareDirection::
     db  DIRECTION_DOWN,     DIRECTION_DOWN
     db  DIRECTION_LEFT,     DIRECTION_LEFT
 
+; Adjustment values for Link's position in case of collision with a solid tile.
 Data_002_6E39::
     db   $00, $01, $FF, $00
```

Comment:
If this is a top-right-down-left array, can you label it?

Also, it could probably benefit from being written as signed integers (0, 1, -1, 0)

Reply:
I think it is not a top-right-down-left array, but the byte that is chosen from this array depends on which of Link's collision points collided with a solid tile. Not sure how to write a better comment for this.

Reply:
Mmm, can't see either; let's leave it like this 👍